provision.sh

```diff
@@ -2,15 +2,23 @@
 set -e
 set -u
 set -o pipefail
+set -x
 
 # Install tools
 sudo mv /tmp/tools/* /usr/local/bin
 
+# TODO: uncomment; commented bc of this error: `NOCHANGE: partition 1 could only be grown by 1919 [fudge=2048]`
 # Grow the filesystem
-sudo growpart /dev/sda 1
-sudo xfs_growfs /
+#sudo growpart /dev/sda 1
+#sudo xfs_growfs /
```

Comment:
@sallyom We can remove those since it is fixed on cloud images.

Reply:
oh great, phew, I thought I saw that, but wasn't sure :)